dotCMS/src/main/java/com/dotmarketing/db/LockTask/LockTask.java

```diff
@@ -0,0 +1,73 @@
+package com.dotmarketing.db.LockTask;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class LockTask implements Serializable {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)